tests/test_flows.py

```diff
@@ -810,58 +846,83 @@ def test_combined_infra_metrics(self):
         vap1.disassociate(sta1)
         vap2.disassociate(sta2)
 
-    def test_client_capability_query(self):
-        sta1 = env.Station.create()
-        sta2 = env.Station.create()
+    def base_test_client_capability_query(self, sta):
```

Comment:
*Nitpick*: please document arguments types. Makes usage of this method easier for others.